README.md

```diff
@@ -126,37 +131,39 @@ Refer to [consensus node documentation](docs/consensus.md).
 
 ## Smart contract development
 
-Please refer to [neo-go smart contract development
+Please refer to [NeoGo smart contract development
```

Comment:
If we're fixing this, then maybe we'd like to fix the `UserAgentPrefix = "NEO-GO:"` as far?

Reply:
I'm scared to even try changing, it was like that since the beginning.